testing/src/cypress/page-objects/ministry-objects/InstitutionNoteRestrictionsObject.ts

```diff
@@ -134,4 +126,28 @@ export default class InstitutionNoteRestrictionsObject {
   generalTabButton() {
     return cy.get(".text-center > .float-right > :nth-child(2)");
   }
+
+  noteDropdown() {
+    return cy.xpath("//div[@role='combobox']");
+  }
+
+  noteValue() {
+    return cy.xpath("//input[@role='textbox']");
+  }
+
+  restrictionsDropdownReason() {
+    return cy.xpath("//div[@id='etm8wyi']//div[@role='combobox']");
```

Comment:
Should I assume that these are not part of the new development and are shown here because of the files reorganization?

Reply:
Yes.